rfcs/models/identity/identity-update-policy.md

```diff
@@ -0,0 +1,69 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# Network Identity Update Policies
+
+* Authors: Bishakh Chandra Ghosh, Venkatraman Ramakrishna, Krishnasuri Narayanam, Ermyas Abebe
+* Status: Proposed
+* Since: 24-September-2021
+
+## updatePolicy
+
+The memberships in a blockchain network are not guaranteed to be static. Often existing participant members of a network leave while new participants join. As a result, the `verificationMethod` and `networkParticipants` of the *Network DID* must also be updated to capture the updated network structure.
+
+Based on the different governance policies of different networks, they might want to enforce different policies that dictate which combination of exisiting parcitipants of the network are allowed to introduce a new member or expel existing members. These policicies can be encoded into the `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*.
+
+`updatePolicy` defines a combination of participant units that can sign and authorize a Network DID updation request. This `updatePolicy` has the condition format as specified in [`verifiablecondition2021`](https://w3c.github.io/did-spec-registries/#verifiablecondition2021) verification method. It can be used to combine different verification methods together to create complex conditional expressions such as logical operations, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
+
+
+Example of `updatePolicy`. 
+```json
+{
+  "id": "did:<iin_name>:<network_name>#updatepolicy",
+  "controller": "did:<iin_name>:<network_name>",
+  "type": "VerifiableCondition2021",
+  "conditionAnd": [{
+      "id": "did:<iin_name>:<network_name>#updatepolicy-1",
+      "controller": "did:<iin_name>:<network_name>",
+      "type": "VerifiableCondition2021",
+      "conditionOr": ["did:<iin_name>:<network_participant_3>#key1",
+        "did:<iin_name>:<network_participant_2>#key3"
+      ]
+    },
+    "did:<iin_name>:<network_participant_1>#key1"
+  ]
+}
```

Comment:
is this expressive enough to:
1. specify a threshold of network participants e.g. N/3 ... similar to the default behaviours if an updated policy is not specified 
2. express different policies for different actions .. joining/leaving etc.

Reply:
1. Yes
2. No - one way to support this would be to specify more policy properties in the verification method such as 'joiningUpdatePolicy', 'leavingUpdatePolicy', etc..

Reply:
For now we are keeping a single `updatePolicy`